org.eclipse.scanning.event/src/org/eclipse/scanning/event/remote/_RunnableDevice.java

```diff
@@ -97,10 +96,10 @@ public void validate(M model) throws ValidationException {
 		} catch (ValidationException ve) {
 			throw ve;
 		} catch (Exception ne) {
-            throw new ValidationException(ne);
+			throw new ValidationException(ne);
 		}
 	}
-	
+
 	@Override
 	public Object validateWithReturn(M model) throws ValidationException {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.eclipse.scanning.api.ValidationException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)